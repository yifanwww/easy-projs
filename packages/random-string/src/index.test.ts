import * as package_exports from './index';

it('exports from this package', () => expectSnapshot(Object.keys(package_exports)));

it('Test class `RandomGenerator`', () => {
    const rg = new package_exports.RandomGenerator();

    const str = rg.string(16);
    expect(str.length).toBe(16);
});
