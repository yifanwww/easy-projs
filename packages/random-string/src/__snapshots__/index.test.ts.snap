// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`exports from this package 1`] = `
Array [
  "RandomGenerator",
]
`;
